scripts/apps/search/components/fields/associatedItems.tsx

```diff
@@ -1,33 +1,33 @@
 import React from 'react';
 import {IArticle} from 'superdesk-interfaces/Article';
 
-const TYPES_MAPPING = {
-    picture: 'photo',
-    graphic: 'graphic',
-    video: 'video',
-    audio: 'audio',
+const TYPES_TO_ICONS = {
+    picture: 'icon-photo',
+    graphic: 'icon-graphic',
+    video: 'icon-video',
+    audio: 'icon-audio',
 };
 
 /**
  * Render icon with count for each item type present in associations.
  */
 export function associatedItems({item}: {item: IArticle}) {
     const associations = Object.values(item.associations || {});
-    const icons = Object.keys(TYPES_MAPPING).map((type) => ({
+    const icons = Object.keys(TYPES_TO_ICONS).map((type) => ({
         type: type,
-        icon: TYPES_MAPPING[type],
+        icon: TYPES_TO_ICONS[type],
```

Comment:
Will it work given the entire classname? Only the part after `icon-` used to be passed before.

Reply:
yep the keys didn't change, just values

Reply:
Right, I didn't notice that `icon` value is not passed anywhere outside and only used in this file.